src/cli.ts

```diff
@@ -150,6 +150,14 @@ const build = debounce(async function build() {
       warnings.push(err);
     };
 
+    if (process.env.SOURCE_DATE_EPOCH) {
+      const ts = +process.env.SOURCE_DATE_EPOCH;
+      if (ts !== Math.floor(ts)) {
+        fail(`SOURCE_DATE_EPOCH value ${process.env.SOURCE_DATE_EPOCH} is not valid`);
+      }
```

Comment:
Is this meant to allow the empty string, and "Infinity", and `1e3`, and `0xF`?

Reply:
It doesn't allow the empty string (it won't pass the check above), but I suppose I may as well guard against the others. Updated.

Reply:
`+'' !== Math.floor(+'')` is false

Reply:
Yeah but so is `if ('')`.